src/create-ssr-algolia-client.ts

```diff
@@ -39,7 +39,7 @@ export function createSSRSearchClient({
   makeStateKey,
 }) {
   const client = algoliasearch(appId, apiKey, {});
-  client.addAlgoliaAgent(`angular-instantsearch ${VERSION}`);
+  client.addAlgoliaAgent(`angular-instantsearch (${VERSION})`);
```

Comment:
We can add a mention that it's on the server.

See https://github.com/algolia/react-instantsearch/pull/2209#pullrequestreview-217245480.